packages/coverage/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@betterer/coverage",
+  "description": "A betterer test which ensures that test coverage increases over time",
+  "version": "5.1.6",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "author": "Lucas Treffenstädt <lucas.treffenstaedt@check24.de>",
+  "homepage": "https://phenomnomnominal.github.io/betterer",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/phenomnomnominal/betterer.git"
+  },
+  "bugs": {
+    "url": "https://github.com/phenomnomnominal/betterer/issues"
+  },
+  "scripts": {
+    "compile": "tsc -b .",
+    "api": "api-extractor run --local --verbose"
+  },
+  "engines": {
+    "node": ">=12"
+  },
+  "dependencies": {
+    "@betterer/betterer": "^5.1.6",
+    "@betterer/constraints": "^5.0.0",
+    "@betterer/logger": "^5.1.5",
+    "@betterer/errors": "^5.0.0",
+    "fp-ts": "^2.11.8",
```

Comment:
`io-ts` and `fp-ts` seems like pretty heavy dependencies to add to the project for this. I wonder if custom JSON validation would be good enough?

Reply:
I wanted to avoid writing a bunch of what amounts to boilerplate code, but yes, I can do it manually.